token/cli/src/main.rs

```diff
@@ -2583,23 +2597,34 @@ fn main() -> Result<(), Error> {
         );
         let websocket_url = solana_cli_config::Config::compute_websocket_url(&json_rpc_url);
 
-        let (signer, fee_payer) = signer_from_path(
-            matches,
-            matches
-                .value_of("fee_payer")
-                .unwrap_or(&cli_config.keypair_path),
-            "fee_payer",
-            &mut wallet_manager,
-        )
-        .map(|s| {
-            let p = s.pubkey();
-            (s, p)
-        })
-        .unwrap_or_else(|e| {
-            eprintln!("error: {}", e);
-            exit(1);
-        });
-        bulk_signers.push(signer);
+        // If the fee_payer is specified, or if the default is used and the file exists, then load it
+        let keypair_path = {
+            if let Some(fee_payer) = matches.value_of("fee_payer") {
+                Some(fee_payer.to_string())
+            } else if std::fs::metadata(&cli_config.keypair_path).is_ok() {
+                Some(cli_config.keypair_path.clone())
+            } else {
+                None
+            }
+        };
+
+        let (signer, fee_payer) = if let Some(keypair_path) = keypair_path {
+            signer_from_path(matches, &keypair_path, "fee_payer", &mut wallet_manager)
+                .map(|s| {
+                    let p = s.pubkey();
+                    (Some(s), Some(p))
+                })
+                .unwrap_or_else(|e| {
+                    eprintln!("error: {}", e);
+                    exit(1);
+                })
+        } else {
+            (None, None)
+        };
+
+        if let Some(signer) = signer {
+            bulk_signers.push(signer);
+        }
```

Comment:
nit: This can get simplified, since `signer` is never used outside. Note: this may not compile -- you may need to pull out the pubkey before the call to `push()`, not sure
```suggestion
        let fee_payer = if let Some(keypair_path) = keypair_path {
            let signer = signer_from_path(matches, &keypair_path, "fee_payer", &mut wallet_manager)
                .unwrap_or_else(|e| {
                    eprintln!("error: {}", e);
                    exit(1);
                });
            bulk_signers.push(signer);
            Some(signer.pubkey())
        } else {
            None
        };
```